<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SWMTool</class>
 <widget class="QWidget" name="SWMTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SWMTool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="hLayout">
     <item>
      <widget class="QLabel" name="lblMCU">
       <property name="text">
        <string>型号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbMCU">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <item>
        <property name="text">
         <string>SWM181</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SWM190</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SWM201</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SWM211</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SWM220</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SWM260</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SWM320</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SWM341</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="hSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblPack">
       <property name="text">
        <string>封装：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbPack">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linFreq">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>40</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabMain">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabPin">
      <attribute name="title">
       <string>管脚分配</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabCAN">
      <attribute name="title">
       <string>CAN 波特率</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblCANBaud">
           <property name="text">
            <string>期望波特率（Kbps）：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="linCANBaud">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>100</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hSpacerC1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lblCANSamp">
           <property name="text">
            <string>采样点（%）：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="linCANSamp">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>75</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hSpacerC2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnCANGen">
           <property name="text">
            <string>生成配置</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="txtCANShow">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSDR">
      <attribute name="title">
       <string>SDRAM 参数</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="lblSDRChip">
           <property name="text">
            <string>器件型号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbSDRChip">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hSpacerSDR">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnSDRGen">
           <property name="text">
            <string>生成配置</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="txtSDRShow">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabJPG">
      <attribute name="title">
       <string>JPEG2Code</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblJPGFile">
           <property name="text">
            <string>JPEG文件：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="linJPGFile"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnJPGFile">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lblJPGOut">
           <property name="text">
            <string>输出格式：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbJPGOut">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>不解码</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RGB565</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BGR565</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>XRGB888</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>XBGR888</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="hSpacerJPG">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnJPGConv">
           <property name="text">
            <string>转换</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="txtJPGShow">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabWav">
      <attribute name="title">
       <string>Wave2Code</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="lblWavFile">
           <property name="text">
            <string>Wave文件：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="linWavFile"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnWavFile">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="lblWavObit">
           <property name="text">
            <string>输出位宽：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbWavObit">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>8-bit</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16-bit</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="hSpacerWav">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="chkWavAll">
           <property name="text">
            <string>同目录下所有Wave文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnWavConv">
           <property name="text">
            <string>转换</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="txtWavShow">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
